specs/destination_test.go

```diff
@@ -174,10 +178,12 @@ spec:
 `,
 		"",
```

Comment:
Would be nice to give these tests some descriptive names :)

Reply:
This isn't the test name but rather the expected error (which is "")

Reply:
Ah okay, sorry, misread it! That's why I think it's better for readability to keep the field labels though 😅